addons/im_livechat/static/src/core/common/chat_window_patch.xml

```diff
@@ -8,6 +8,7 @@
             </t>
         </xpath>
         <xpath expr="//Composer" position="replace">
+            <t t-if="thread?.composerDisabled and !store.self.eq(thread?.livechatVisitorMember?.persona)">$0</t>
```

Comment:
```suggestion
            <t t-if="thread?.composerDisabled and store.self.notEq(thread?.livechatVisitorMember?.persona)">$0</t>
```

Reply:
Could we refactor this to avoid the `double $0`? I don't think it's a good thing.

Reply:
> Could we refactor this to avoid the `double $0`? I don't think it's a good thing.

This doesn't change much, could have been
```xml
<t t-if="!thread?.composerDisabled or store.self.notEq(thread?.livechatVisitorMember?.persona)">$0</t>
<t t-if="thread?.composerDisabled">
   ...
</t>
```

Diff is bigger, resulting code is slightly shorter. That's really not important.